src/ucs/type/status.c

```diff
@@ -62,6 +62,8 @@ const char *ucs_status_string(ucs_status_t status)
         return "User-defined limit was reached";
     case UCS_ERR_UNSUPPORTED:
         return "Unsupported operation";
+    case UCS_ERR_PENDING:
```

Comment:
maybe define it in UCP as (UCS_ERR_LAST-1) since this is not a generic error

Reply:
yep, re-defined